src/shared/components/case-editor/case-edit-form/case-edit-form.html

```diff
@@ -1,6 +1,6 @@
 <ng-container *ngFor="let field of fields">
     <div ccdConditionalShow ccdLabelSubstitutor [caseField]="field" [formGroup]="formGroup" [eventFields]="eventFields">
-      <ng-container [ngSwitch]="field | ccdIsReadOnly">
+      <ng-container [ngSwitch]="field | ccdIsReadOnlyAndNotCollection ">
```

Comment:
Cannot we have `ccdIsReadOnly` in place and a new pile being just `ccdIsNotCollection` and combine the two like `field | ccdIsReadOnly | ccdIsNotCollection`?

Reply:
By doing that we would not be repeating `ccdIsReadOnly` inside `ccdIsReadOnlyAndNotCollection` I think.

Reply:
Even though the field is marked as READONLY on the BE (since the user doesn't have UPDATE right), we want to handle COLLECTION fields differently and let the user create new collection items if the user has CREATE right. So on this step we don't want to treat collection as a `<ccd-field-read>` even though the field is set as READ only by the BE.

Reply:
Ok all clear now.